lib/Extension/ClassMover/Application/ClassMover.php

```diff
@@ -82,11 +85,11 @@ public function moveFile(ClassMoverLogger $logger, string $filesystemName, strin
     private function doMoveFile(ClassMoverLogger $logger, string $filesystemName, string $srcPath, string $destPath): void
     {
         $filesystem = $this->filesystemRegistry->get($filesystemName);
-        $destPath = Phpactor::normalizePath($destPath);
         if (str_ends_with($destPath, '/')) {
-            $destPath = $destPath . basename($srcPath);
+            $destPath .= basename($srcPath);
         }
 
+        $destPath = Phpactor::normalizePath($destPath);
```

Comment:
Moving `Phpactor::normalizePath` down because it removes trailing slashes. The correct solution would probably to check: `is_dir($destPath)`.